memoize/memoized.py

```diff
@@ -17,6 +17,7 @@
 import tempfile
 import inspect
 
+
```

Comment:
This commit should be merged to the previous one and then `push -f` to the branch.
It's true that it can be squashed when the branch is accepted, but it's a bad practice and makes reviewing annoying.

Reply:
Can you think of a simple way to automatically check this? 
It would be nice to avoid generating and deleting multiple files.

Reply:
Not sure what you mean - you can run `black` locally and verify before asking for a PR - am I missing something?

Reply:
I don't understand your last comment about the linter. Linting is Ok for this commit. 

But: I do want to add the test. First I want to rebase this branch, so it includes example etc. How?